src/main/java/fi/otavanopisto/kuntaapi/server/controllers/ServiceController.java

```diff
@@ -62,19 +62,30 @@ public Service findService(ServiceId serviceId) {
     
     return null;
   }
-  
-  public List<Service> listServices(OrganizationId organizationId, Long firstResult, Long maxResults) {
-    List<Service> result = new ArrayList<>();
-    
-    for (ServiceProvider serviceProvider : getServiceProviders()) {
-      result.addAll(serviceProvider.listServices(organizationId));
-    }
-    
-    return ListUtils.limit(entityController.sortEntitiesInNaturalOrder(result), firstResult, maxResults);
-  }
+//  
+//  public List<Service> listServices(OrganizationId organizationId, Long firstResult, Long maxResults) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)